### PR TITLE
updates on screenshots and circulars

### PR DESCRIPTION
**Pull request by jracusin**
updates with screenshots of current development and updated summary of circulars to reflect what will be in initial release

### CODE REVIEW

**Comment by jracusin** [comment]
Also included a slide about the new notice producers documentation

**Comment by jracusin** [comment]
Screenshot 2023-03-05 at 1 00 13 PM

File: index.html [hunk 1]
```diff
@@ -622,11 +622,13 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
     <section>
       <div class="image-and-caption">
-        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
+        <img src="img/screen-shot-07-step-4.png" width="850" alt="Screen shot">
         <div data-markdown>
           <textarea data-template>
             ## Step 4:<br>Get Sample Code
-            Copy and paste Python client code or download it to your computer to run.
+            Copy and paste Python client code or download it to your computer to run. <br>
+            <br>
```
Comment (lpsinger) [inline index.html:630]: This is Markdown. Create paragraphs by skipping a line.

```suggestion
            Copy and paste Python client code or download it to your computer to run.

```

File: index.html [hunk 2]
```diff
@@ -622,11 +622,13 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+            Client code sample also availble in Node.js (ESM), Node.js (Common.JS), C/C++, C#.
```
Comment (lpsinger) [inline index.html:631]: Fix several spelling errors (available, CommonJS):

```suggestion
            Client sample code is also available in Node.js (ESM or CommonJS), C/C++, C#.
```

File: index.html [hunk 3]
```diff
@@ -507,12 +507,12 @@ <h2>Why switch to the new GCN?</h2>
...
-        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol)
+        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol) of all current notice types
         * GCN Circulars submission and delivery via email
         * The old GCN Classic web site, https://gcn.gsfc.nasa.gov
-        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) and [GCN Circulars](https://gcn.gsfc.nasa.gov/gcn3_archive.html) on the old web site
-
-        However, starting later this year new features and notice types will only be available on the new web site and GCN Kafka.
+        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site
+      
+        New features and notice types are only available on the new web site and GCN Kafka
```
Comment (lpsinger) [inline index.html:515]:
```suggestion
        However, new features and notice types are only available on the new web site and GCN Kafka.
```

File: index.html [hunk 4]
```diff
@@ -622,11 +622,13 @@ <h1>Streaming GCN<br>Notices in Python</h1>
 
     <section>
       <div class="image-and-caption">
-        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
+        <img src="img/screen-shot-07-step-4.png" width="850" alt="Screen shot">
```
Comment (lpsinger) [inline index.html:625]:
```suggestion
        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
```

I'll supply new screen shots that match the dimensions of the original ones.

File: index.html [hunk 5]
```diff
@@ -682,6 +684,31 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+</section>
+
+<section>
+  <div class="image-and-caption">
+    <div>
+      <img src="img/screen-shot-new-producers.png" width="800" alt="Screen shot of New Notice Producers Documentation">
+    </div>
+    <div data-markdown>
+      <textarea data-template>
+        ## Distributing New Notice Types
```
Comment (lpsinger) [inline index.html:699]:
```suggestion
        ## Create New Notice Types
```

File: index.html [hunk 6]
```diff
@@ -682,6 +684,31 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+        * All new notice topics will only be distributed by GCN Kafka
+        * [Step-by-step instruction](https://gcn.nasa.gov/docs/producers)
```
Comment (lpsinger) [inline index.html:701]:
```suggestion
        * See [step-by-step instructions](https://gcn.nasa.gov/docs/producers)
```

File: index.html [hunk 7]
```diff
@@ -745,6 +854,7 @@ <h1>Streaming GCN<br>Notices in Python</h1>
           }
         }
         ```
+-->
```
Comment (lpsinger) [inline index.html:857]: We're using version control! Don't comment it out, delete it!

File: index.html [hunk 8]
```diff
@@ -716,7 +766,7 @@ <h1>Streaming GCN<br>Notices in Python</h1>
         </div>
       </div>
     </section>
-
+  -->
```
Comment (lpsinger) [inline index.html:769]: Don't comment it out, delete it!

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: index.html (hunk 8 above)
Comment (jracusin) [reply index.html:769]: I was trying to see if I could fit the additional enhancement text, but ok.

File: index.html [hunk 9]
```diff
@@ -507,12 +507,12 @@ <h2>Why switch to the new GCN?</h2>
...
         GCN Classic is not going away any time soon. The following are still fully supported:
-        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol)
+        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol) of all current notice types
         * GCN Circulars submission and delivery via email
         * The old GCN Classic web site, https://gcn.gsfc.nasa.gov
-        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) and [GCN Circulars](https://gcn.gsfc.nasa.gov/gcn3_archive.html) on the old web site
-
-        However, starting later this year new features and notice types will only be available on the new web site and GCN Kafka.
+        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site
+
```
Comment (lpsinger) [inline index.html:514]:
```suggestion

```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -507,12 +507,11 @@ <h2>Why switch to the new GCN?</h2>
...
         ## What’s staying the same?
         GCN Classic is not going away any time soon. The following are still fully supported:
-        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol)
+        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol) of all current notice types
         * GCN Circulars submission and delivery via email
         * The old GCN Classic web site, https://gcn.gsfc.nasa.gov
-        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) and [GCN Circulars](https://gcn.gsfc.nasa.gov/gcn3_archive.html) on the old web site
-
-        However, starting later this year new features and notice types will only be available on the new web site and GCN Kafka.
+        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site
```
Comment (lpsinger) [inline index.html:513]:
```suggestion
        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site

```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: index.html [hunk 11]
```diff
@@ -687,35 +689,30 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
-      </div>
-    </section>
+<section>
+  <div class="image-and-caption">
+    <div>
+      <img src="img/screen-shot-new-producers.png" width="800" alt="Screen shot of New Notice Producers Documentation">
+    </div>
+    <div data-markdown>
+      <textarea data-template>
+        ## Create New Notice Types
```
Comment (lpsinger) [inline index.html:707]: For the most part, we aren't using title case for slide headings.

```suggestion
        ## Create new Notice types
```

File: index.html [hunk 12]
```diff
@@ -507,12 +507,12 @@ <h2>Why switch to the new GCN?</h2>
...
-        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol)
+        * GCN Notices legacy delivery mechanisms (email, socket, VOEvent Transport Protocol) of all current notice types
         * GCN Circulars submission and delivery via email
         * The old GCN Classic web site, https://gcn.gsfc.nasa.gov
-        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) and [GCN Circulars](https://gcn.gsfc.nasa.gov/gcn3_archive.html) on the old web site
-
-        However, starting later this year new features and notice types will only be available on the new web site and GCN Kafka.
+        * The live archives of [GCN Notices]
+        (https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site
+        However, new features and notice types are only available on the new web site and GCN Kafka.
```
Comment (lpsinger) [inline index.html:515]:
```suggestion
        * The live archives of [GCN Notices](https://gcn.gsfc.nasa.gov/burst_info.html) on the old web site

        However, new features and notice types are only available on the new web site and GCN Kafka.
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: index.html [hunk 13]
```diff
@@ -727,27 +724,69 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+            ## Modernization of GCN Circulars (Coming Soon!)
+            * Manage your own subscriptions
+            * Self-register to submit using peer endorsement
+            * Submit circulars by web form, API, or email
+            * New searchable archive of Circulars
+            * Planned enhancements
+              * "Astro-Flavored Markdown"
+              * Automatically mint DOIs and BiBTeX entries
+              * Link your ORCID to your GCN Circulars
+
```
Comment (lpsinger) [inline index.html:783]:
```suggestion
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]

**Pull request merged**